src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/Json/DateTimeOffsetNullableConverter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class DateTimeOffsetNullableConverter : JsonConverter
+    {
```

Comment:
looked into it--turns out it's unnecessary to have this, as `Newtonsoft.Json` is smart enough to convert it without this custom converter.